components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/model/ExtendedUser.java

```diff
@@ -0,0 +1,103 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+package org.wso2.carbon.identity.mgt.endpoint.util.client.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ExtendedUser {
```

Comment:
Why can't we use the existing user object

Reply:
This backend supports an additional attribute 'isUsernameCaseSensitive' related to code, that is not present in the current User object. Currently, this attribute is not used, but the username validations against code can honor this attribute. This is exposing all the details provided from the backend in this API.